src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,84 @@
+package duke;
+
+/**
+ * Parser for commands
+ */
+public class Parser {
+
+    private String listALl(TaskList tl) {
```

Comment:
I think there there's a typo in the method name! `listAL1` isn't really clear.

Reply:
Thanks for spotting this!